contracts/introspection/SupportsInterfaceWithLookup.sol

```diff
@@ -9,6 +9,8 @@ import "./ERC165.sol";
  * @dev Implements ERC165 using a lookup table.
  */
 contract SupportsInterfaceWithLookup is ERC165 {
+
+  // solium-disable-next-line uppercase
```

Comment:
I'd consider either removing this rule from our ruleset, or enforcing it. IMO it makes no sense to require constants to be uppercase, only to later disable it. @elopio WDYT?

Reply:
we _do_ manually disable it in a lot of places (ERC20 as well in `name` and `symbol`). I'd support either disabling it or keeping it as-is (warning).

Reply:
Hm, I think it'd nuke it then. I'm against warnings that will go ignored.